Configuration/HLT/python/autoCondHLT.py

```diff
@@ -1,5 +1,7 @@
 # https://cms-conddb.cern.ch/browser/#search
 
+snapshotTime = "9999-12-31 23:59:59.000"
```

Comment:
Also, the previous code is a no-op: when you define a customisation of the GT, the default snapshot time is set to "infinity": see [here](https://github.com/cms-sw/cmssw/blob/CMSSW_8_0_X/CondCore/ESSources/plugins/CondDBESSource.cc#L150-L173).